.github/workflows/pull_request.yml

```diff
@@ -41,9 +28,27 @@ jobs:
           reporter: github-pr-review
           path: '.'
           pattern: '*.sh'
+          
+      - uses: cachix/install-nix-action@v22
+      - name: Check if the chart follows a valid semver version
+        run: |
+          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' scripts/nix/sources.json -r)
+          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+          branch_name="${{ github.event.pull_request.base.ref }}"
+          nix-shell --pure --run "./scripts/validate-version.sh --branch $branch_name" ./scripts/shell.nix
 
-
-  unit-test:
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(nix-shell --pure --run "ct list-changed --config ct.yaml" ./scripts/shell.nix)
```

Comment:
what does this changed signify here?

Reply:
It just checks if the chart has changed or not, so that we can conditionally run the lint, might not be needed. Seems I missed adding the condition in the lint stage.

Reply:
Any downside to always just running the lint?

Reply:
I don't think so, I will remove this stage.